x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml

```diff
@@ -181,3 +190,63 @@ setup:
           {
             "fixed_interval": "1h"
           }
+
+---
+"Rollup not time_series index":
+  - skip:
+      version: " - 8.4.99"
+      reason: "rollup renamed to downsample in 8.5.0"
+
+  - do:
+      indices.create:
+        index: normal-index
+
+  - do:
+      catch: /Rollup requires setting \[index.mode=time_series\] for index \[normal-index\]/
+      indices.downsample:
+        index: normal-index
+        target_index: rollup-test
+        body:  >
+          {
+            "fixed_interval": "1h"
+          }
+
+
+---
+"Rollup no metric index":
```

Comment:
I think a `skip` section is needed here.

Reply:
done